packages/bot/src/struct/CommandHandler.ts

```diff
@@ -169,6 +211,25 @@ export class CommandHandler {
 			const mod = (await import(pathToFileURL(file).toString())) as { default: CommandConstructor };
 			const command = container.resolve(mod.default);
 
+			if (command.containsSubcommands) {
```

Comment:
I'm not fond of this folder structure after all. I would like
```
- commands
  - someNormalCommand.ts
  - commandWithSubCommands
    - index.ts (this is where the CommandWithSubcommands is exported from)
    - someSubcommand.ts
    - someOtherSubcommand.ts
```

Reply:
so is the `commandWithSubCommands` folder supposed to be the name of the command that has subcommands, the index file exports the `commandWithSubcommand` implementation for that command, and then the subcommands are also within that folder?

Reply:
yup!